app.js

```diff
@@ -29,13 +33,25 @@ app.use(require('node-sass-middleware')({
 }));
 app.use(express.static(path.join(__dirname, 'public')));
 
-// app.use('/', routes);
+app.use(require('express-session')({
+  secret: 'Brenna is Bae',
```

Comment:
i'm assuming we don't want people to know what this is, so this would mean making it an environment variable (like the mongoURI)